### PR TITLE
Improve usability of popover 

### PR DESCRIPTION
**Pull request by thangdinh**
Basically, the idea is to apply the "tab inside modal" to popover:
+ Allow tab only inside popover 
+ Avoid using global key handler to handle ESC key.

@Addepar/pretzel @Addepar/uie @igillis

### CODE REVIEW

File: src/popover.coffee [hunk 1]
```diff
@@ -45,6 +45,12 @@ Ember.Widgets.BodyEventListener,
 
   didInsertElement: ->
     @_super()
+
+    # Make sure that after the popover is rendered, set focus to the first
+    # tabbable element
+    Ember.run.schedule 'afterRender', this, ->
```
Comment (igillis) [inline src/popover.coffee]: Looks like this is duplicated here and in modal. Could we move it into `TabbableModal`? Not sure in what order the `didInsertElement` hooks get run.

**Comment by igillis** [comment]
lgtmbc

File: src/popover.coffee (hunk 1 above)
Comment (thangdinh) [inline src/popover.coffee]: Good point. I just refactored it.

**Comment by embooglement** [comment]
@thangdinh can we close this?

**Comment by embooglement** [comment]
Closing this for now. Feel free to reopen.

**Pull request closed without merging**